### PR TITLE
style bugs graph update

### PR DESCRIPTION
**Pull request by jomongeorge**

### CODE REVIEW

File: client/src/main.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ import App from './App.vue'
 import router from './router'
 import store from './store'
 import './filters'
-
+import * as nem2Sdk from 'nem2-sdk'
```
Comment (AnthonyLaw) [inline client/src/main.js]: I dont think we need this for here, we can import directly to file when we need it.

File: client/src/main.js [hunk 2]
```diff
@@ -37,15 +37,16 @@ Vue.component('page-footer', Pagefooter)
 Vue.component('time-since', TimeSince)
 Vue.component('loader', w1)
 Vue.use(VueClipboard)
+Vue.use(nem2Sdk)
```
Comment (AnthonyLaw) [inline client/src/main.js]: we can import directly to file when we need it.

Comment (jomongeorge) [reply client/src/main.js]: yea I put this here for an experiment , removed it

File: client/src/main.js (hunk 1 above)
Comment (jomongeorge) [reply client/src/main.js]: removed import * as nem2Sdk from 'nem2-sdk'

**Pull request merged**